lib/adapters/http/index.js

```diff
@@ -950,7 +950,7 @@ function HttpPouch(opts, callback) {
             if (returnDocs) {
               results.results.push(c);
             }
-            utils.call(opts.onChange, c);
+            opts.onChange(c);
```

Comment:
whenever we know that the function is not undefined, I'd prefer to just call it directly than use `utils.call()`, which is a weird function anyway.